### PR TITLE
[fuchsia][cml] Mark Tracing as Optional

### PR DESCRIPTION
**Pull request by gmittert**
Tracing is an optional component that is only included in eng builds, so the usage should be marked optional.

Bug: fxbug.dev/112433

### CODE REVIEW

**Comment by gmittert** [comment]
First time having to PR to flutter, let me know if I missed anything in the contributing guidelines.

Internal staged presubmits: cl/547246543

**Review by chinmaygarde — APPROVED** [review]

**Comment by gmittert** [comment]
@chinmaygarde I don't have merge access, so I'll need someone to merge this for me once the tree is no longer broken.

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Review by arbreng — APPROVED** [review]

**Pull request merged**
